v1/atm/atm.gunk

```diff
@@ -157,19 +160,24 @@ type ATMService interface {
 	// }
 	GetATM(GetATMRequest) ATM
 
-	// GetATMs get all the available ATMs.
+	// GetATMs retrieves information regarding all the available ATMs.
 	//
 	// +gunk http.Match{
 	//         Method: "GET",
 	//         Path:   "/v1/atms",
 	// }
 	// +gunk openapiv2.Operation{
 	//         Tags:        []string{"ATMs"},
-	//         Description: "Get All available ATMs.",
-	//         Summary:     "Get All available ATMs.",
+	//         Description: "Retrieves information regarding all the available ATMs.",
+	//         Summary:     "Retrieve all available ATMs",
 	//         Responses: map[string]openapiv2.Response{
 	//                 "200": openapiv2.Response{
 	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/atmGetATMsResponse",
```

Comment:
Should the Ref be "#/definitions/atm`s`GetATMsResponse"?

Reply:
same as https://github.com/openbank/openbank/pull/17#discussion_r314579305